### PR TITLE
Adds Salt Overdose Effects

### PR DESCRIPTION
**Pull request by nullbear**
lol

:cl:
rscadd: New research shows that eating too much salt makes you salty.
/:cl:

### CODE REVIEW

**Comment by nullbear** [comment]
@Iamgoofball huge innovative chemistry change get it while its hot and fresh.

**Comment by kevinz000** [comment]
...

**Comment by nullbear** [comment]
I spent a lot of time working on this.

**Comment by optimumtact** [comment]
Did I ever tell you... what the definition of insanity is?

**Comment by JJRcop** [comment]
Meme pr?

**Comment by Kromgar** [comment]
So do you intend to have people turn into pillar of salts at the end of overdose?

**Comment by nullbear** [comment]
It's not even so bad that it IS a terrible meme pr.

**Comment by bgobandit** [comment]
third time's a charm

File: code/modules/reagents/chemistry/reagents/food_reagents.dm [hunk 1]
```diff
@@ -246,6 +246,15 @@
 	description = "A salt made of sodium chloride. Commonly used to season food."
 	reagent_state = SOLID
 	color = "#FFFFFF" // rgb: 255,255,255
+	overdose_threshold = 30
+
+/datum/reagent/consumable/sodiumchloride/overdose_process(mob/living/M)
+	var/high_message = pick("You feel really upset.", "You feel really cranky.", "You feel like crying.", "You feel like shouting about how unfair your life is.")
+	if(prob(10))
```
Comment (MrStonedOne) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: Please put the message inside this if. No need to do a pick operation if you're gonna ignore it 90% of the time

**Comment by optimumtact** [comment]
anyway no, this has been denied many times before, stop making pr's for it.

**Comment by PKPenguin321** [comment]
Actually when it was denied that was because it was as strong as acid, this is completely different

**Comment by MrStonedOne** [comment]
>Actually when it was denied that was because it was as strong as acid, this is completely different

**Comment by yackemflam** [comment]
Proof that maintainers don't even maintain

File: code/modules/reagents/chemistry/reagents/food_reagents.dm (hunk 1 above)
Comment (nullbear) [inline code/modules/reagents/chemistry/reagents/food_reagents.dm]: Done! But the commit will only take effect if the PR is reopened.

**Pull request closed without merging**